### PR TITLE
Update layout.jade

### PR DESCRIPTION
**Pull request by dhairya2019**
Code changed for linking shopping-server/public/stylesheets/style.css inside the shopping-server/views/layout.jade

### CODE REVIEW

File: shopping-server/views/layout.jade [hunk 1]
```diff
@@ -2,6 +2,6 @@ doctype html
 html
   head
     title= title
-    link(rel='stylesheet', href='/stylesheets/style.css')
+    link(rel='stylesheet', href='../public/stylesheets/style.css')
```
Comment (smaharj1) [inline shopping-server/views/layout.jade:5]: You do not need to do this. It is automatically connected with https://github.com/Viveckh/Veniqa/blob/develop/shopping-server/app.js#L87

File: shopping-server/package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "jsonwebtoken": "^8.4.0",
     "lodash": "^4.17.11",
     "mongodb": "^3.1.10",
-    "mongoose": "^5.3.15",
+    "mongoose": "^5.9.12",
```
Comment (smaharj1) [inline shopping-server/package.json:33]: There is already another PR out with this change. Thank you.

File: shopping-server/views/layout.jade (hunk 1 above)
Comment (dhairya2019) [reply shopping-server/views/layout.jade:5]: Ok I got it. Thanks:)

File: shopping-server/package.json (hunk 2 above)
Comment (dhairya2019) [reply shopping-server/package.json:33]: Okay

**Comment by smaharj1** [comment]
Please look at the current Issues board and feel free to pick any of them. Thanks a lot!

**Pull request closed without merging**